### ISSUE TITLE
[New Badge Request]

### ISSUE THREAD

**Issue by Sang-Buster** [Envoy-VC/awesome-badges#26]
- Badge name: Matlab
- Category: IDE or Analytics 

![image](https://user-images.githubusercontent.com/97267956/196054214-862a88c3-3f2b-471a-8488-e8232088ed10.png)

**Comment by Envoy-VC** [comment]
@Sang-Buster Sorry for delay, it's done.

**Comment by Sang-Buster** [comment]
You're fine, thanks for the creation!! 😁👍 Although this image is not in a `shield.io/badge` link, I am wondering if you mind make the text color white, thanks!

**Comment by Sang-Buster** [comment]
@Envoy-VC , I notice the Matlab badge has a copy right, and the one you have it inked is just shows as a raw github content link, you might want to take it down, or link the below image link to fit with the shield.io badge style. The below image I created it by converting MATLAB logo into base 64 code and pasting it inside the svg file. 

![](https://raw.githubusercontent.com/Sang-Buster/Picgo-Github/main/img/Icons/Matlab.svg)

**Comment by Envoy-VC** [comment]
Hey @Sang-Buster  I got the Badge for MALAB but like its made of inline base64 svg

![MATLAB](https://img.shields.io/static/v1?message=R2022b&label=MATLAB&color=orange&style=for-the-badge&logo=data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciICB2aWV3Qm94PSIwIDAgNDggNDgiIHdpZHRoPSI0OHB4IiBoZWlnaHQ9IjQ4cHgiPjxsaW5lYXJHcmFkaWVudCBpZD0iWjhiRzg5VG5aVzh+QndKanpxbW5YYSIgeDE9IjIyLjY0NSIgeDI9IjI2Ljc1NyIgeTE9IjEwLjg4MSIgeTI9IjIzLjg1NCIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiPjxzdG9wIG9mZnNldD0iMCIgc3RvcC1jb2xvcj0iIzRhZGRkZiIvPjxzdG9wIG9mZnNldD0iLjY5OSIgc3RvcC1jb2xvcj0iIzNmNTM1MiIvPjxzdG9wIG9mZnNldD0iLjg2MyIgc3RvcC1jb2xvcj0iIzQ0MjcyOSIvPjwvbGluZWFyR3JhZGllbnQ+PHBhdGggZmlsbD0idXJsKCNaOGJHODlUblpXOH5Cd0pqenFtblhhKSIgZD0iTTIxLDI3bC03LTZjMCwwLDEtMS41LDIuNS0zczIuNzM2LTEuODUyLDQuNS0zYzMuNTExLTIuMjg0LDYuNS0xMiwxMS0xMkwyMSwyN3oiLz48bGluZWFyR3JhZGllbnQgaWQ9Ilo4Ykc4OVRuWlc4fkJ3Smp6cW1uWGIiIHgxPSIxIiB4Mj0iMzcuNzc1IiB5MT0iMjcuMDMzIiB5Mj0iMjcuMDMzIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+PHN0b3Agb2Zmc2V0PSIwIiBzdG9wLWNvbG9yPSIjNGFkZGRmIi8+PHN0b3Agb2Zmc2V0PSIuNzkyIiBzdG9wLWNvbG9yPSIjM2Y1MzUyIi8+PHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjNDQyNzI5Ii8+PC9saW5lYXJHcmFkaWVudD48cG9seWdvbiBmaWxsPSJ1cmwoI1o4Ykc4OVRuWlc4fkJ3Smp6cW1uWGIpIiBwb2ludHM9IjExLDMzLjA2NiAxLDI2IDE0LDIxIDIxLjI3NywyNi40NjUgMTQsMzIuMDY2Ii8+PGxpbmVhckdyYWRpZW50IGlkPSJaOGJHODlUblpXOH5Cd0pqenFtblhjIiB4MT0iMTEiIHgyPSI0NyIgeTE9IjI0IiB5Mj0iMjQiIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIj48c3RvcCBvZmZzZXQ9Ii4yMDYiIHN0b3AtY29sb3I9IiM1MzE0MGYiLz48c3RvcCBvZmZzZXQ9Ii4zIiBzdG9wLWNvbG9yPSIjODQzNjBmIi8+PHN0b3Agb2Zmc2V0PSIuNDEzIiBzdG9wLWNvbG9yPSIjYjg1YjEwIi8+PHN0b3Agb2Zmc2V0PSIuNTExIiBzdG9wLWNvbG9yPSIjZGY3NjEwIi8+PHN0b3Agb2Zmc2V0PSIuNTkiIHN0b3AtY29sb3I9IiNmNjg3MTAiLz48c3RvcCBvZmZzZXQ9Ii42MzkiIHN0b3AtY29sb3I9IiNmZjhkMTAiLz48c3RvcCBvZmZzZXQ9Ii43MjkiIHN0b3AtY29sb3I9IiNmZDhhMTAiLz48c3RvcCBvZmZzZXQ9Ii44IiBzdG9wLWNvbG9yPSIjZjU4MDEwIi8+PHN0b3Agb2Zmc2V0PSIuODY1IiBzdG9wLWNvbG9yPSIjZTg2ZjEwIi8+PHN0b3Agb2Zmc2V0PSIuOTI1IiBzdG9wLWNvbG9yPSIjZDY1ODExIi8+PHN0b3Agb2Zmc2V0PSIuOTgyIiBzdG9wLWNvbG9yPSIjYzAzYTExIi8+PHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjYjcyZjExIi8+PC9saW5lYXJHcmFkaWVudD48cGF0aCBmaWxsPSJ1cmwoI1o4Ykc4OVRuWlc4fkJ3Smp6cW1uWGMpIiBkPSJNMzIsM2M1LDAsMTMsMjcsMTUsMzRjMCwwLTcuMDE3LTYuNjMtMTEtNnMtNS40Nyw2LjU0OC05LjcyNSwxMC43NTZDMjMuNSw0NC41LDIxLDQ1LDIxLDQ1CXMtMC4yMDYtOC4xMjQtNS0xMWMtMi41LTEuNS01LTEtNS0xczYuMDQ5LTIuOTAxLDkuNDc0LTguMTc0UzI4LjUsMywzMiwzeiIvPjwvc3ZnPg==)

**Comment by Sang-Buster** [comment]
Right, this is what I had to do also (see below svg file) ,
![](https://raw.githubusercontent.com/Sang-Buster/Picgo-Github/main/img/Icons/Matlab.svg)  
If you wish to include it, I made a badge along with MATLAB's color, if you want to put it in the `assets` folder? 

I was just suggesting deleting the `https://user-images.githubusercontent.com/97267956/196054214-862a88c3-3f2b-471a-8488-e8232088ed10.png` in your README file since MATLAB is a proprietary software and its logo has a copyright. Anyway, thanks for the feedback and taking the time to make it! 😄👍
